### ISSUE TITLE
Is await really needed?

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/react-invenio-deposit#61]
Here there is an await: https://github.com/inveniosoftware/react-invenio-deposit/pull/60/files#diff-b5b3deb83dcaa18c58f0a85d23df3e40R51 but there is also in the create function that it calls. I wonder if it is really needed.

**Comment by qusaifares** [comment]
Yes, `this.createDraft()` returns a promise and you must use await for it to resolve and return the actual data.
